Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Collections/GridObjectCollection.cs

```diff
@@ -0,0 +1,419 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Internal.Extensions;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.Collections
+{
+    /// <summary>
+    /// A Grid Object Collection is simply a set of child objects organized with some
+    /// layout parameters.  The collection can be used to quickly create 
+    /// control panels or sets of prefab/objects.
+    /// </summary>
+    public class GridObjectCollection : BaseObjectCollection
+    {
+        #region public accessors
+        [Tooltip("Type of surface to map the collection to")]
+        private ObjectOrientationSurfaceTypeEnum surfaceType = ObjectOrientationSurfaceTypeEnum.Plane;
```

Comment:
Please ensure you use the correct spacings in the script. 
Properties both public and private should be spaced between them.